src/_defaults.scss

```diff
@@ -120,13 +120,40 @@ h1, .h1, h2, .h2, h3, .h3, h4, .h4, h5, .h5, h6, .h6
 	font-weight: $bold-weight;
 }
 
-.h3,
-.h4,
-.h5,
-.h6 {
+.h3:not(summary),
+.h4:not(summary),
+.h5:not(summary),
+.h6:not(summary) {
 	border: 0;
 }
```

Comment:
IMO these selectors are really strange :S. They might even be pointless...

I only kept them in place out of an abundance of caution since I'm trying to sneak in this PR at the last minute.

Those selectors only target a subset of heading classes and leave their equivalent heading elements alone. That causes deviations between the heading classes and elements. Plus neither Bootstrap 3.4.x, WET 4.x nor GCWeb contain any selectors that attempt to setup borders on H2-H6 headings.

FWIW these selectors were first introduced in #33 and #868. The former seems to indicate it was related to IA spec changes (possibly something to do with GCWeb's old mega menu or mobile menu). No idea if they still serve any useful purpose these days.